storage/innobase/xtrabackup/doc/source/installation/compiling_xtrabackup.rst

```diff
@@ -31,37 +31,62 @@ In Debian-based distributions, you need to:
 
 .. code-block:: bash
 
- $ apt-get install build-essential flex bison automake autoconf \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
-    libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
+   $ apt install build-essential flex bison automake autoconf \
+   libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
+   libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
 
-
-In ``RPM``-based distributions, you need to:
+In ``RPM``-based distributions, you need to install and enable `the
+Developer Toolset 7
+<https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/>`_ to
+make sure that you use the latest GCC compiler and development tools. |Percona Xtrabackup|
+requires GCC version 5.3 or higher. Then, install ``cmake3`` and other dependencies:
```

Comment:
same for cmake. fedora and rhel 8 do not have package `cmake3`, `cmake` needs to be installed. Those who will copy-paste entire `yum` command will simply get an error message...